lib/wanda/messaging/adapters/amqp/publisher.ex

```diff
@@ -16,7 +18,7 @@ defmodule Wanda.Messaging.Adapters.AMQP.Publisher do
   def publish_message(message, routing_key \\ "") do
     Logger.info("Publishing message #{inspect(message)}")
 
-    GenRMQ.Publisher.publish(__MODULE__, message, routing_key)
+    GenRMQ.Publisher.publish(__MODULE__, message, routing_key, [{:content_type, Contracts.content_type}])
```

Comment:
`Contracts.content_type` is a function, right? Don't you need to finish with `()`?
I think at least `Credo` should complain about

Reply:
You're right, the rubyst in me take over everything 🤣

Reply:
fixed, make ruby great again! 🤣